presentation/src/main/java/com/yapp/presentation/ui/member/signup/Name.kt

```diff
@@ -0,0 +1,190 @@
+        modifier = Modifier.fillMaxSize()
+    ) {
+        if (showDialog) {
+            YDSPopupDialog(
+                title = stringResource(id = R.string.name_cancel_dialog_title),
+                content = stringResource(id = R.string.name_cancel_dialog_subtitle),
+                negativeButtonText = stringResource(id = R.string.name_cancel_dialog_no),
+                positiveButtonText = stringResource(id = R.string.name_cancel_dialog_cancel),
+                onClickPositiveButton = { onClickBackBtn() },
+                onClickNegativeButton = { showDialog = !showDialog },
+                onDismiss = { showDialog = !showDialog }
+            )
+        }
+
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            Column(
+                modifier = Modifier.padding(horizontal = 24.dp)
+            ) {
+                Title()
+                InputName(
+                    name = uiState.name,
+                    onInputName = { viewModel.setEvent(NameContract.NameUiEvent.InputName(it)) }
+                )
+            }
+            NextButton(
+                name = uiState.name,
+                isKeyboardOpened = isKeyboardOpened,
+                modifier = Modifier.align(Alignment.BottomCenter),
+                onClickNextBtn = onClickNextBtn
+            )
+        }
+    }
+}
+
+@Composable
+fun Title() {
+    Column() {
+
+        Spacer(modifier = Modifier.height(32.dp))
+        Text(
+            text = stringResource(id = R.string.name_title),
+            color = Color.Black,
+            style = AttendanceTypography.h1
+        )
+        Spacer(modifier = Modifier.height(12.dp))
+        Text(
+            text = stringResource(id = R.string.name_subtitle),
+            color = Gray_800,
+            style = AttendanceTypography.body1
+        )
+        Spacer(modifier = Modifier.height(28.dp))
+    }
+}
+
+@Composable
+fun InputName(name: String, onInputName: (String) -> Unit) {
+
+    BasicTextField(
+        value = name,
+        onValueChange = { onInputName(it); },
+        singleLine = true,
+        textStyle = AttendanceTypography.body2.copy(color = Gray_800),
+        cursorBrush = SolidColor(Yapp_Orange),
+        decorationBox = { innerTextField ->
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(color = Gray_200, shape = RoundedCornerShape(50.dp))
+            ) {
+                Box(modifier = Modifier.padding(20.dp)) {
+                    if (name.isBlank()) {
+                        Text(
+                            text = stringResource(id = R.string.name_example_hint),
+                            color = Gray_400,
+                            style = AttendanceTypography.body1
+                        )
+                    } else {
+                        innerTextField()
+                    }
+                }
+            }
+        }
+    )
```

Comment:
```suggestion
    TextField(
        value = name,
        onValueChange = { onInputName(it); },
        singleLine = true,
        modifier = Modifier
            .fillMaxWidth()
            .background(color = Gray_200, shape = RoundedCornerShape(50.dp)),
        placeholder = {
            Text(
                text = stringResource(id = R.string.name_example_hint),
                color = Gray_400,
                style = AttendanceTypography.body1
            )
        },
        textStyle = AttendanceTypography.body2.copy(color = Gray_800),
        colors = TextFieldDefaults.textFieldColors(
            textColor = Color.Gray,
            disabledTextColor = Color.Transparent,
            backgroundColor = Gray_200,
            focusedIndicatorColor = Color.Transparent,
            unfocusedIndicatorColor = Color.Transparent,
            disabledIndicatorColor = Color.Transparent
        )
    )
```

이렇게 사용해보시겠어요? 잘 되는 것 같네요!